be/src/common/config.h

```diff
@@ -240,6 +240,7 @@ CONF_mInt32(disk_stat_monitor_interval, "5");
 CONF_mInt32(profile_report_interval, "30");
 CONF_mInt32(unused_rowset_monitor_interval, "30");
 CONF_String(storage_root_path, "${STARROCKS_HOME}/storage");
+CONF_Bool(encrypted, "false");
```

Comment:
Is the naming of this configuration too simple and I can’t tell what is being encrypted?

Reply:
by requirement, all user data on disk should be encrypted, currently including segment/delete files, pk index, datacache,  will add binlog/fulltext index in following PR, probably will add spill output in future.

Reply:
how about enable_transparent_data_encryption?